sites/public/layouts/application.tsx

```diff
@@ -36,44 +36,43 @@ const Layout = (props) => {
           logoSrc="/images/logo_glyph.svg"
           notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
           title={t("nav.siteTitle")}
-          languages={LANGUAGES}
+          languages={languages}
         >
-          <LocalizedLink href="/listings" className="navbar-item">
-            {t("nav.listings")}
-          </LocalizedLink>
+          <Link href="/listings">
```

Comment:
Feels a little weird to see a <Link> with a `href` wrapping an `a` tag without one, but I don't necessarily have a suggestion

Reply:
That's just how Next's link component works — although because I had to add a special `LinkComponent` wrapper for pushing into the `NavigationContext` for the components layer, it's possible we might decide to use `LinkComponent` ourselves too. But that'd be a "non-standard" pattern as far as Next is concerned.